### ISSUE TITLE
Missing **Android** in header

### ISSUE THREAD

**Issue by wung-s** [RepairShopr/react-native-signature-capture#63]
Were it not pointed out to me by my colleague, I would have totally missed out on the Android support. I remember seeing the header description and moving on because I needed one that had support for both the platform (iOS & Android) but the header seemed to suggest support only for iOS. 

![signature-capture](https://user-images.githubusercontent.com/5159585/27207790-019288a8-525f-11e7-95e1-441d5b373ef8.png)

**Comment by spinningarrow** [comment]
Yes, almost made this mistake as well. Can the repo description be updated?
